pages/about.js

```diff
@@ -0,0 +1,112 @@
+//global imports
+import React from "react";
+import Layout from "../src/components/layouts/aboutLayout";
+import { Typography, Container } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+//data
+import info from "../public/data/info.json";
+import members from "../public/data/members.json";
+import links from "../public/data/links.json";
+//local components
+import AboutHeaderImage from "../src/components/about/AboutHeaderImage";
+import InfoBubble from "../src/components/about/InfoBubble";
+import InfoLink from "../src/components/about/InfoLink";
+import Member from "../src/components/about/Member";
+
+const backgroundImage = `url("images/about_background.png")`;
+const useStyles = makeStyles(theme => {
+  return {
+    root: {
+      textAlign: "center"
+    },
+    textBlock: {
+      paddingTop: theme.spacing(2),
+      paddingBottom: theme.spacing(1)
+    },
+    sectionHeadline: {
+      marginBottom: theme.spacing(6)
+    },
+    bubbleGrid: {
+      padding: 0,
+      maxWidth: 1390,
+      margin: "0 auto",
+      textAlign: "center"
+    },
+    memberGrid: {
+      maxWidth: 1390,
+      margin: "0 auto",
+      textAlign: "center",
+      verticalAlign: "top",
+      padding: 0
+    },
+    mainFocuses: {
+      width: 800,
+      margin: "0 auto"
+    }
+  };
+});
+
+export default function Home() {
+  const classes = useStyles();
+  return (
+    <>
+      <Layout>
+        <AboutHeaderImage image={{ url: backgroundImage }} />
+        <Typography variant="h4" color="primary" className={`${classes.root} ${classes.textBlock}`}>
```

Comment:
This can look like an `h4` but should be an `h1` for accessibility. Can we add the `component="h1"` prop to this?

```jsx
<Typography variant="h4" component="h1" ...
```